rest-server/data/ur.json

```diff
@@ -3,21 +3,22 @@
   "engineFactory": "com.actionml.engines.ur.UREngine",
   "sparkConf": {
     "master": "yarn",
+    "appName": "harness-spark-test-app",
```

Comment:
this should be derived from the engine-id so it is guaranteed to be unique across all jobs by all engines.